lib/src/shared/message.dart

```diff
@@ -69,8 +70,8 @@ StreamTransformer<GrpcMessage, GrpcMessage> grpcDecompressor() =>
         handleData: (GrpcMessage value, EventSink<GrpcMessage> sink) {
       if (value is GrpcData) {
         if (value.isCompressed) {
```

Comment:
This is cool, but will not work on for grpc_web. That doesn't have access to dart:io.